### PR TITLE
Added morphs(WIP)

### PR DESCRIPTION
**Pull request by Wouterz90**
Still needs icons/tooltips/sounds

### CODE REVIEW

**Comment by darklordabc** [comment]
Things todo:

- [x] Add the transfer sound. We can use the normal sound that gets used in the original strength morph, not sure what its called. 

- [x] I changed the shift rate, to what it is in the original morphing, but its different to what you had. You had stats per tick, but the original showed tick rate, I changed it to the tick rate so the ability does not work right now. 
image

@Wouterz90

**Comment by Wouterz90** [comment]
It's untested, might get to that later.

File: src/localization/addon_english.txt [hunk 1]
```diff
@@ -9216,6 +9216,44 @@
         "DOTA_Tooltip_ability_bellatrix_Unholy_Ecstasy_damage_cap"                                  "DAMAGE CAP:"
         "DOTA_Tooltip_ability_bellatrix_Unholy_Ecstasy_buff_duration"                               "BUFF DURATION:"
     
+<<<<<<< HEAD
```
Comment (darklordabc) [inline src/localization/addon_english.txt]: This isnt right

File: src/localization/addon_english.txt [hunk 2]
```diff
@@ -9216,6 +9216,44 @@
...
+
+        "DOTA_Tooltip_Unholy_Ecstasy_Bonus_Damage"                                        "Unholy Ecstasy"
+        "DOTA_Tooltip_Unholy_Ecstasy_Bonus_Damage_Description"                            "Your next attack deals extra damage based on the damage that was dealt to you during Unholy Ecstasy"
+        
+        
+        
+        
+
+
+=======
```
Comment (darklordabc) [inline src/localization/addon_english.txt]: Nor that

File: src/localization/addon_english.txt [hunk 3]
```diff
@@ -9267,6 +9305,7 @@
 
     "DOTA_Tooltip_modifier_morph_int_agi"              "Morph (INT to AGI)"
     "DOTA_Tooltip_modifier_morph_int_agi_Description"      "Converting Intelligence into Agility."
+>>>>>>> 709ad62333ab74e0db373790328ce975d3c3d706
```
Comment (darklordabc) [inline src/localization/addon_english.txt]: Or this

**Comment by darklordabc** [comment]
Seems to be working fine

**Pull request merged**